realtime/src/services/exchanges/free-currency-rates/index.ts

```diff
@@ -0,0 +1,125 @@
+import axios from "axios"
+
+import {
+  InvalidTickerError,
+  ExchangeServiceError,
+  UnknownExchangeServiceError,
+  InvalidExchangeConfigError,
+} from "@domain/exchanges"
+import { toPrice, toSeconds, toTimestamp } from "@domain/primitives"
+import { LocalCacheService } from "@services/cache"
+import { CacheKeys } from "@domain/cache"
+import { baseLogger } from "@services/logger"
+
+export const FreeCurrencyRatesExchangeService = async ({
+  base,
+  quote,
+  config,
+}: FreeCurrencyRatesExchangeServiceArgs): Promise<
+  IExchangeService | ExchangeServiceError
+> => {
+  if (!config) return new InvalidExchangeConfigError()
+
+  const { baseUrl, fallbackUrl, timeout, cacheSeconds } = config
+
+  const url =
+    baseUrl || "https://cdn.jsdelivr.net/gh/fawazahmed0/currency-api@1/latest/currencies/"
+  const cacheKey = `${CacheKeys.CurrentTicker}:freecurrencyrates:${base}:*`
+
+  const getCachedRates = async (): Promise<FreeCurrencyRatesRates | undefined> => {
+    const cachedTickers = await LocalCacheService().get<FreeCurrencyRatesRates>(cacheKey)
+    if (cachedTickers instanceof Error) return undefined
+    return cachedTickers
+  }
+
+  const fetchTicker = async (): Promise<Ticker | ServiceError> => {
+    // We cant use response.data.date because
+    // FreeCurrencyRates does not behave like a bitcoin exchange
+    const timestamp = new Date().getTime()
+    const baseCurrency = base.toLowerCase()
```

Comment:
Do we need this lower-casing in the `currencybeacon` service (or any others) too?

Reply:
no, is because in this case  base currency is part of the returned json but in lower case (in currency beacon is only passed in query param)